### PR TITLE
feat: update available icons dynamically

### LINKED ISSUE #33: [Feat]: Update available icons dynamically
### Feature request

Icons can be dynamically found [here](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/config/links.json). I noticed your version was slightly behind or didn't include all of them. You could gather them here and generate the options this way. It would be a cool feature. I could build it if you'd like a PR.

### PR DESCRIPTION
**Pull request by loftwah**
PR is ready for the start of Hacktoberfest.

resolve #33

### CODE REVIEW

**Comment by loftwah** [comment]
BTW I don't really care about counting stuff for Hacktoberfest, I intend on doing heaps anyway :)

File: package.json [hunk 1]
```diff
@@ -25,4 +25,4 @@
   "dependencies": {
     "prettier": "^2.7.1"
   }
-}
+}
```
Comment (schmelto) [inline package.json:28]:
```suggestion
}

```

File: js/scripts.js [hunk 2]
```diff
@@ -150,64 +150,47 @@ async function appVersion() {
...
-                <option value="YouTube">YouTube</option>
-            </select>
-        </td>
-        <td><button class="btn btn-sm btn-danger ibtnDel">Delete</button></td>
-        `;
-    newRow.append(cols);
-    $("table.order-list").append(newRow);
-    counter++;
-  });
+
```
Comment (schmelto) [inline js/scripts.js:173]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]
Some small things

File: js/scripts.js [hunk 3]
```diff
@@ -221,41 +204,8 @@ $(document).ready(function () {
...
-                            <option value="Send">Send</option>
-                            <option value="Slack">Slack</option>
-                            <option value="Snapchat">Snapchat</option>
-                            <option value="Telegram">Telegram</option>
-                            <option value="TikTok">TikTok</option>
-                            <option value="Twitch">Twitch</option>
-                            <option value="Twitter">Twitter</option>
-                            <option value="Vimeo">Vimeo</option>
-                            <option value="YouTube">YouTube</option>
+                            <! Loop through the icons array and create an option for each icon using getIcons() !>
```
Comment (schmelto) [inline js/scripts.js:207]: I think here the implementation is missing

Comment (loftwah) [reply js/scripts.js:206]: Good pickup, I ignored the milestone section.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

I will wait until hacktoberfest start than I will merge the changes. 

Thanks for your contribution!

---
**Summary:** :ok_hand: This code looks good

**Pull request merged**